src/main/java/br/com/shapeup/common/exceptions/post/PostNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package br.com.shapeup.common.exceptions.post;
+
+public class PostNotFoundException extends RuntimeException {
+    public PostNotFoundException(String id) {
+        super("Post " + id + " not found");
```

Comment:
Só um detalhe, é mais por frescur aminha. Da para usar o `String.format("Post %s not found", id)` acho que fica mais elegantee

Reply:
papo retissímo, e deixa mais padronizado também, vou alterar